event-hubs.api.md

```diff
@@ -0,0 +1,303 @@
+## API Report File for "@azure/event-hubs"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AbortSignalLike } from '@azure/abort-controller';
+import { AmqpError } from 'rhea-promise';
+import { AwaitableSender } from 'rhea-promise';
+import { ConnectionContextBase } from '@azure/core-amqp';
+import { DataTransformer } from '@azure/core-amqp';
+import { DefaultDataTransformer } from '@azure/core-amqp';
+import { Dictionary } from 'rhea-promise';
+import { EventHubConnectionConfig } from '@azure/core-amqp';
+import { MessagingError } from '@azure/core-amqp';
+import { Receiver } from 'rhea-promise';
+import { ReceiverOptions } from 'rhea-promise';
+import { RetryOptions } from '@azure/core-amqp';
+import { SharedKeyCredential } from '@azure/core-amqp';
+import { Span } from '@azure/core-tracing';
+import { SpanContext } from '@azure/core-tracing';
+import { SpanOptions } from '@azure/core-tracing';
+import { TokenCredential } from '@azure/core-amqp';
+import { TokenType } from '@azure/core-amqp';
+import { WebSocketImpl } from 'rhea-promise';
+
+// @public
+export interface AbortSignalOptions {
```

Comment:
I dont believe we need a separate interface for AbortSignalOptions. Based on usage in other libraries, the `abortSignal`  stands alone. Having a dedicated interface like this that shows up in docs without providing much of a value doesnt look right.

@bterlson Any reason why you would want to have a dedicated type for abort signal options?

Reply:
Is abort signal the same as CloseReason? Or Closecontext? I can't tell.

Reply:
It's akin to CancellationToken in C# (not sure what the equivalent is in Java)

Reply:
Interface flattened

Reply:
I still see it ...

Reply:
Reverted your last commit that caused all changes to go back to the first version of this PR